pkg/crc/machine/machine.go

```diff
@@ -195,11 +196,32 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.InfoF("Starting OpenShift cluster ... [waiting 3m]")
 	}
 	result.KubeletStarted = kubeletStarted
-	//
 
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
+		// Checks after 3 mins wait all the operators are in 'Available' state.
+		// If any of the operator still not available then wait for 2 more mins and tell users
+		// about which operator is still not available and need time.
+		var status bool
+		for i := 0; i < 2; i++ {
+			status, err = oc.GetClusterOperatorStatus(filepath.Join(constants.MachineCacheDir, "oc", "oc"), filepath.Join(extractedPath, crcBundleMetadata.ClusterInfo.KubeConfig))
+			if err != nil {
+				logging.ErrorF("Failed to get cluster status: %v", err)
+				result.Error = err.Error()
+				return *result, err
+			}
+			if status {
+				continue
+			}
+			time.Sleep(time.Minute)
+		}
+
+		if !status {
+			logging.Warn("Mentioned cluster operators are still not in completely 'Available' state. Wait for some more time before using this cluster")
```

Comment:
this suggestion does not provide enough info to what the user should look for with this command.

it is better to expose this as a possible status command (to concise the info) and allow for a verbose option to get all details.